airflow/serialization/serialized_objects.py

```diff
@@ -1318,6 +1318,8 @@ def serialize_task_group(cls, task_group: TaskGroup) -> dict[str, Any] | None:
         # when converting set to list, the order is uncertain.
         # When calling json.dumps(self.data, sort_keys=True) to generate dag_hash, misjudgment will occur
         encoded = {
+            "is_setup": task_group.is_setup,
+            "is_teardown": task_group.is_teardown,
```

Comment:
Seems like we no longer use is_setup/is_teardown on task_group?

Reply:
yeah can be removed